src/service/url-expander/expander.js

```diff
@@ -86,13 +88,14 @@ export class Expander {
     let matchIndex = 0;
     let match = matches[matchIndex];
     let numOfPendingCalls = 0;
+    let ignoringChars = false;
+    let nextArgShouldBeRaw = false;
 
     const evaluateNextLevel = () => {
       let builder = '';
       const results = [];
 
       while (urlIndex < url.length && matchIndex <= matches.length) {
-
         if (match && urlIndex === match.start) {
```

Comment:
Why can we get a match in the middle of a literal string?

Reply:
If I understand your question correctly, this is to support things like `example.com?cid=CLIENT_ID(__ga)`

Reply:
No, I mean why would `CLIENT_ID` match inside of

```
cid=`CLIENT_ID(__GA)`
```

the tick marks a literal string ("don't parse me" section).

Reply:
the way it was designed is so that the ticks are only respected inside a macro call (inside parenthesis). We are currently reserving keywords like `CLIENT_ID` to be replaced anywhere in the input string.  We could potentially change the parser to look for this if you think its a valid use case?